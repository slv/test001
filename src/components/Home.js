import React from "react";
import { Home } from "./Home.theme";

export const Component = () => <Home>Homepage</Home>;

Component.displayName = "Home";

export default Component;
