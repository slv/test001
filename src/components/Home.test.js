import React from "react";
import ReactDOM from "react-dom";
import renderer from "react-test-renderer";
import Home from "./Home";

describe("Home", () => {
  it("renders without crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(<Home />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it("match snapshot", () => {
    const renderedValue = renderer.create(<Home />).toJSON();
    expect(renderedValue).toMatchSnapshot();
  });
});
