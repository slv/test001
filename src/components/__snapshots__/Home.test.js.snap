// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home match snapshot 1`] = `
<div
  className="sc-bdVaJa eIxvgw"
>
  Homepage
</div>
`;
